contents/handbook/small-teams/customer-success/objectives.mdx

```diff
@@ -1,10 +1,12 @@
-* **Proposed Objective:**
-    * Scale Customer Success through automation
-* **Proposed Key Results:**
-    * Finish Q4 with 24 more customers in the 20-100k ARR range than we started.
-    * Every customer gets an automated CS intro email tailored based on their Ideal Customer Profile (ICP).
-    * We can easily report on a customer's adoption and react according to where they are on the journey to provide maximum benefit.
-* **Rationale:**
-    * We have learned a lot from selling this year and now it’s time to scale
-    * We are getting more and more signups each week.
-    * We need to improve conversion. Being more effective will come from automation.
+- **Objective 1**: Nail company revenue goals
+  - Reasoning: We are primarily focused on growth and retention and so we should measure ourselves on that.
+    - **KR**: Finish Q1 with 15 more $20k+ ARR customers
+    - **KR**: Add another large customer on an annual plan ($100k+)
+- **Objective 2**: Maintain existing customer base
+  - Reasoning: Looking after our existing customer base is super important for overall revenue growth.
+    - **KR**: No churn in $20k+ ARR customers who’ve paid 2x bills of >$1667
+    - **KR**: Mechanism to measure first pass SLA for all 20K ARR customers regardless of channel
```

Comment:
what is "first pass" SLA?

Reply:
Good point - pushed an edit but it's essentially first response to slack/email queries from 20k ARR customers